src/PKSim.Core/Model/IndividualFactory.cs

```diff
@@ -57,6 +60,10 @@ public Individual CreateAndOptimizeFor(OriginData originData, int? seed = null)
             individual.Seed = seed.Value;
 
          _createIndividualAlgorithm.Optimize(individual);
+
+         var diseaseStateImplementation = _diseaseStateImplementationFactory.CreateFor(individual);
+         diseaseStateImplementation.ApplyTo(individual);
```

Comment:
For a healthy individual it will be a "Do nothing"-implementation, correct?

Reply:
yes